client/forest-manager/src/rocksdb.rs

```diff
@@ -503,6 +503,75 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_generate_proof_includes_neighbor_keys() {
```

Comment:
Do you think it's worth it to create a test or modify this one to include the apply delta execution?

Reply:
Just added it!